actioncable/lib/action_cable/server/configuration.rb

```diff
@@ -28,8 +28,13 @@ def pubsub_adapter
         path_to_adapter = "action_cable/subscription_adapter/#{adapter}"
         begin
           require path_to_adapter
+        rescue Gem::MissingSpecVersionError => e # Introduced in RubyGems 2.6.3
+          # This is raised when `gem 'foo', '~> x.y.z'` couldn't be satisfied,
+          # which happens with adapters' optional dependencies. Bubble up the
+          # actual exception rather than wrapping/obfuscating.
+          raise
```

Comment:
One thing we miss out on here is `#{adapter}` -- it might be useful to know that the complaint is about a dep of the configured ACa adapter.

I'm not sure whether it's worth wrapping the exception in our own (maybe while copying over its backtrace?). The full/framework backtrace will mention this line, and the adapter itself... but our backtrace cleaner will strip that out by default, making it pretty unclear where the dep is coming from.

Reply:
Agree. Took a simpler tack in 4f50849d74a4a208b2832e11c5fc528adea3818b to distinguish missing adapters ("couldn't load adapter, maybe missing from Gemfile") from errors loading an adapter ("Error loading adapter, missing a gem it depends on? <original message>")

Reply:
👌🏻